src/DwitTech.AccountService.Core/Services/EmailService.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using DwitTech.AccountService.Core.Interfaces;
+using DwitTech.AccountService.Core.Models;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using RestSharp;
+
+namespace DwitTech.AccountService.Core.Services
+{
+    public class EmailService : IEmailService
+    {
+
+        private readonly IConfiguration _configuration;
+
+        public EmailService()
+        {
+        }
+
+        public EmailService(IConfiguration configuration)
+        {
+            _configuration = configuration;
+            
+        }
+        public async Task<bool> SendMailAsync(string userId, string fromEmail, string toEmail, string subject, string body, string cc, string bcc)
+        {
+            var emailObject = new Email { FromEmail = fromEmail, ToEmail = toEmail, Subject = subject, Body = body };
```

Comment:
If you do it like this, then what will happen to bc and cc? I'm confused by this. pleaseexplain.